manim/mobject/coordinate_systems.py

```diff
@@ -310,6 +311,13 @@ def init_background_lines(self):
         )
 
     def get_lines(self):
+        """Generate all the lines, faded and not faded. Two sets of lines are generated, parallel to the X-axis and to Y-axis.
```

Comment:
Could you change the wording up a little here:
Generate all the lines, faded and not faded. Two sets of lines are generated: one parallel to the X-axis, and parallel to the Y-axis.